sdk/cosmos/azure-cosmos/samples/container_management.py

```diff
@@ -181,6 +181,19 @@ def create_container(db, id):
     except exceptions.CosmosResourceExistsError:
         print('A container with id \'_container_analytical_store\' already exists')
 
+    print("\n2.8 Create Container - With auto scale settings")
+
+    try:
+        container = db.create_container(
+            id=id+"_container_auto_scale_settings",
+            partition_key=partition_key,
+            offer_throughput=Offer(auto_scale_max_throughput=5000, auto_upgrade_throughput_increment_percent=0)
```

Comment:
We shouldn't use the object `Offer` here - that was deprecated in favor of `ThroughputProperties`.
Also - I think the param name is `auto_scale_increment_percentage`?

Reply:
We cannot reach the object without exposing **Offer** so what should we do? Do we expose it anyway? currently we are not passing in **ThroughputProperty** in our __init__ file.

Reply:
Then that would seem to be a bug that should be fixed. :)

Reply:
so for this PR in particular do I still go with Offer or change it to ThroughputProperty and see about it being fixed after?